p_soc_auto_base/models.py

```diff
@@ -40,7 +40,7 @@ class BaseModel(models.Model):
         _('created on'), db_index=True, auto_now_add=True,
         help_text=_('object creation time stamp'))
     updated_on = models.DateTimeField(
-        _('updated on'), db_index=True, auto_now_add=True,
+        _('updated on'), db_index=True, auto_now_add=False,
```

Comment:
this needs to be auto_now=True (a different property). don't worry about it; it's fixed somewhere else